signer/local/local.go

```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/cloudflare/cfssl/info"
 	"github.com/cloudflare/cfssl/log"
 	"github.com/cloudflare/cfssl/signer"
+
+	"encoding/asn1"
+	"encoding/binary"
```

Comment:
final nits: "encoding/asn1" and "encoding/binary" should be grounded with standard libraries above.